backend-api/src/functions/asyncToken/asyncTokenHandler.ts

```diff
@@ -38,15 +36,15 @@ export async function lambdaHandlerConstructor(
     logger.log("INVALID_REQUEST", {
       errorMessage: eventBodyResult.value.errorMessage,
     });
-    return badRequestResponseInvalidGrant;
+    return badRequestResponse("Invalid grant type or grant type not specified");
```

Comment:
I almost think this would look nice as a builder. So you don't have to scroll down to work out what badRequestResponse is. 

Not for now but would be curious for your thoughts as a talking point